### PR TITLE
feat(curriculum): Bash Commands quiz

### LINKED ISSUE #56230: [HACKTOBERFEST 2024] - Bash Commands and Command Line Quiz
> [!IMPORTANT]
> This issue is part of our 2024 Hacktoberfest initiative. If you are interested in working on this issue, refer to our [top-level tasks](https://github.com/freeCodeCamp/freeCodeCamp/issues/56170) to see if this issue is claimed, or to claim it if it is not.

This issue is to create the Bash Commands and Command Line quiz.

The file you should be working on is found in `/curriculum/challenges/english/25-front-end-development/quiz-bash-commands`.

**All quiz questions need to relate to one of these topics:**

- Terminal vs. command line
- Short-cuts in command line to speed up your workflows
- bash scripting and some basic commands
- command options/flags
- navigating the fs in the terminal
- One of the courses before this quiz is [located here.](https://github.com/freeCodeCamp/learn-bash-by-building-a-boilerplate/blob/main/TUTORIAL.md) Any commands or topics covered in that course can be used in the quiz. Here's a partial list: `pwd`, `ls`, `cd`, `mv`, `rm`, `cp`, `mkdir`, `touch`, `--help` flag, `find`, `rmdir`, recursive flag, 

> [!CAUTION]
> Questions which do not fall under these topics will be rejected.

You are welcome to include code blocks in your questions, but please keep them short (four lines or less).

Please do NOT open a pull request until you have completed all questions. Your quiz should have **exactly 20 questions**.

### PR DESCRIPTION
**Pull request by shubha987**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #56230 

I have added the quiz from the content provided

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/quiz-bash-commands/66f1af4fedf643c78d024c5e.md [hunk 1]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+What command is used to see the path of working directory?
 
 #### --distractors--
 
-Placeholder distractor 1
+a) `ls`
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/quiz-bash-commands/66f1af4fedf643c78d024c5e.md:24]: do not give letters to the distractors or answers

File: curriculum/challenges/english/25-front-end-development/quiz-bash-commands/66f1af4fedf643c78d024c5e.md [hunk 2]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
 ---
 
-Placeholder distractor 3
+c) `pwd`
 
 #### --answer--
 
-Placeholder answer
+c) `pwd`
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/quiz-bash-commands/66f1af4fedf643c78d024c5e.md:36]: there are 3 distractors and one answer, total 4 choices, and they need to be all different from each other.

File: curriculum/challenges/english/25-front-end-development/quiz-bash-commands/66f1af4fedf643c78d024c5e.md (hunk 1 above)
Comment (shubha987) [reply curriculum/challenges/english/25-front-end-development/quiz-bash-commands/66f1af4fedf643c78d024c5e.md:24]: ok i am fixing it

**Comment by shubha987** [comment]
I have tried to resolve it. Can you please look into it if any changes requires?

File: curriculum/challenges/english/25-front-end-development/quiz-bash-commands/66f1af4fedf643c78d024c5e.md [hunk 3]
```diff
@@ -21,19 +21,19 @@ What command is used to see the path of working directory?
 
 #### --distractors--
 
-a) `ls`
+mv
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/quiz-bash-commands/66f1af4fedf643c78d024c5e.md:24]:
```suggestion
`mv`
```

backticks are still needed

**Comment by shubha987** [comment]
ok i have added backticks to all the options

File: curriculum/challenges/english/25-front-end-development/quiz-bash-commands/66f1af4fedf643c78d024c5e.md [hunk 4]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
-Placeholder answer
+`mkdir`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Print 'I finished the example quiz' to the terminal using command
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-bash-commands/66f1af4fedf643c78d024c5e.md:218]:
```suggestion
Which command would print the text `I finished the example quiz`?
```

File: curriculum/challenges/english/25-front-end-development/quiz-bash-commands/66f1af4fedf643c78d024c5e.md [hunk 5]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
-Placeholder answer
+`echo I finished the example quiz`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which command you will be using if you want to add a text to file directly from terminal?
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-bash-commands/66f1af4fedf643c78d024c5e.md:240]:
```suggestion
Which command would you use if you want to add text a to file directly from the terminal?
```

File: curriculum/challenges/english/25-front-end-development/quiz-bash-commands/66f1af4fedf643c78d024c5e.md [hunk 6]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
-Placeholder answer
+`cp -r <folder_to_copy> <name_of_copy>`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which flag is used with find to search for exact folder?
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-bash-commands/66f1af4fedf643c78d024c5e.md:416]:
```suggestion
Which flag is used with find to search for an exact folder?
```

File: curriculum/challenges/english/25-front-end-development/quiz-bash-commands/66f1af4fedf643c78d024c5e.md [hunk 7]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
 ---
 
-Placeholder distractor 3
+`mv -name <folder>`
 
 #### --answer--
 
-Placeholder answer
+`find -name <folder>`
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-bash-commands/66f1af4fedf643c78d024c5e.md:432]: These answers don't align with the question. The question asks about the flag, not the command.

File: curriculum/challenges/english/25-front-end-development/quiz-bash-commands/66f1af4fedf643c78d024c5e.md [hunk 8]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which flag is used with command `ls` to list the content of a folder in 'long list format;
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-bash-commands/66f1af4fedf643c78d024c5e.md:438]:
```suggestion
Which flag is used with the `ls` command to list the contents of a folder in long list format?
```

File: curriculum/challenges/english/25-front-end-development/quiz-bash-commands/66f1af4fedf643c78d024c5e.md [hunk 9]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+What command is used to see the path of working directory?
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-bash-commands/66f1af4fedf643c78d024c5e.md:20]:
```suggestion
What command is used to see the path of the working directory?
```

File: curriculum/challenges/english/25-front-end-development/quiz-bash-commands/66f1af4fedf643c78d024c5e.md [hunk 10]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
-Placeholder answer
+`ls`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which will be used to rename the 'menlo.font' to 'menlo.otf'
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-bash-commands/66f1af4fedf643c78d024c5e.md:152]:
```suggestion
Which command will rename the 'menlo.font' file to 'menlo.otf'
```

File: curriculum/challenges/english/25-front-end-development/quiz-bash-commands/66f1af4fedf643c78d024c5e.md [hunk 11]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
-Placeholder answer
+`mv menlo.font menlo.otf`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which command you will be used to find things or view a file tree ?
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-bash-commands/66f1af4fedf643c78d024c5e.md:174]:
```suggestion
Which command can be used to find files or view a file tree ?
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
Off to a great start! We have some grammar fixes.

File: curriculum/challenges/english/25-front-end-development/quiz-bash-commands/66f1af4fedf643c78d024c5e.md (hunk 11 above)
Comment (shubha987) [reply curriculum/challenges/english/25-front-end-development/quiz-bash-commands/66f1af4fedf643c78d024c5e.md]: I have tried to resolve a question issue and grammar fixes. If there is any inconsistency let me know again

**Review by naomi-lgbt — APPROVED** [review]
Alright, I think we're good to go! Nice work!

**Review by gikf — APPROVED** [review]
Note for future - it is recommended to make changes on a separate branch, rather than the `main` branch. See more about it in point 3 at https://contribute.freecodecamp.org/how-to-contribute-to-the-codebase/#contributing-to-the-codebase.

**Comment by gikf** [comment]
Hi @shubha987. Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**